lib/services/project-service.ts

```diff
@@ -101,16 +104,25 @@ export class ProjectService implements IProjectService {
 			} else {
 				shell.cp('-R', path.join(appSourcePath, "*"), appDestinationPath);
 			}
-			this.createBasicProjectStructure(projectDir,  projectId).wait();
+			
+			// Copy tns-core-modules package to node_modules destination directory
+			var nodeModulesDestination = path.join(appDestinationPath, constants.NODE_MODULES_FOLDER_NAME);
```

Comment:
See latest commit. It has a check for `tns-core-modules` version. 
There are two options to pass the version number. 
The first one is to use an optional parameter `--tns-modules-version` when executing `create` command. The other possibility is to have defined the version in `package.json` file which comes with the app contents. 
In both cases the intallation of tns_modules happens on `create` phase.